### PR TITLE
feature(Connection): add profile metrics to payload

### PR DESCRIPTION
**Pull request by anta-semenov**
Not sure if data should be optional or not. My thought was that when we just create connection we probably won't have these metrics ready, not sure if I'm right in this assumtion

### CODE REVIEW

**Review by lucjac — APPROVED** [review]
👍

File: src/version-5/social-connections.ts [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
     disconnected = 'disconnected',
 }
 
+interface ConnectionProfileLanguageData {
+  readonly language: LanguageISO
+  readonly learnedWords: number
+}
+
+interface ConnectionProfileData {
+  readonly curretnStreak: number
```
Comment (lucjac) [inline src/version-5/social-connections.ts:16]: typo :)

File: src/version-5/social-connections.ts [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+  readonly streakRecord: number
+  readonly numberOf100PercentAccuracySession: number
```
Comment (lucjac) [inline src/version-5/social-connections.ts:18]: should we rename to numberOf100PercentAccuracySessions? (including s)

File: src/version-5/social-connections.ts (hunk 1 above)
Comment (anta-semenov) [reply src/version-5/social-connections.ts]: thanks, fixed

File: src/version-5/social-connections.ts (hunk 2 above)
Comment (anta-semenov) [reply src/version-5/social-connections.ts]: fixed

**Pull request merged**